backend/dissemination/forms.py

```diff
@@ -14,11 +50,89 @@ class SearchForm(forms.Form):
     end_date = forms.DateField(required=False)
     cog_or_oversight = forms.CharField(required=False)
     agency_name = forms.CharField(required=False)
-    audit_year = forms.MultipleChoiceField(choices=AY_choices, required=False)
+    audit_year = forms.MultipleChoiceField(
+        choices=AY_choices, initial=[2023], required=False
+    )
+    findings = forms.MultipleChoiceField(choices=findings_choices, required=False)
+    direct_funding = forms.MultipleChoiceField(
+        choices=direct_funding_choices, required=False
+    )
+    major_program = forms.MultipleChoiceField(
+        choices=major_program_choices, required=False
+    )
     auditee_state = forms.CharField(required=False)
 
     # Display params
     limit = forms.CharField(required=False)
     page = forms.CharField(required=False)
     order_by = forms.CharField(required=False)
     order_direction = forms.CharField(required=False)
+
+    def clean_aln(self):
+        """
+        Clean up the ALN field. Replace common separators with a newline.
+        Split on the newlines. Strip all the resulting elements.
+        """
+        text_input = self.cleaned_data["aln"]
+        text_input = (
+            text_input.replace(",", "\n")
```

Comment:
Lets pull these out into a list, add \t. 

For loop over the things in both places.